content/en/02-installing-pixie/01-requirements.md

```diff
@@ -5,14 +5,24 @@ metaDescription: "Resource requirements to install Pixie"
 order: 1
 ---
 
-Below are the requirements for deploying Pixie to your Kubernetes (K8s) cluster.
+Before installing Pixie to your Kubernetes cluster, please ensure that your system meets the requirements below.
+
+- [Kubernetes Environment](/installing-pixie/requirements/#kubernetes)
+- [Operating System](/installing-pixie/requirements/#operating-system)
+- [CPU](/installing-pixie/requirements/#cpu)
+- [Network](/installing-pixie/requirements/#network-traffic)
+- [Memory](/installing-pixie/requirements/#memory)
 
 Please refer to the [install guides](/installing-pixie/install-guides/) for information on how to install Pixie to your K8s cluster.
 
 ## Kubernetes
 
 Kubernetes `v1.16+` is required.
 
+The following tables list Kubernetes environments that have been tested with Pixie.
+
+<Alert variant="outlined" severity="info">Pixie may work on other K8s environments. If you find an unlisted K8s environment that works well, please let us know by opening a GitHub issue or by creating a pull request that updates <a href="https://github.com/pixie-io/docs.px.dev/blob/main/content/en/02-installing-pixie/01-requirements.md">this doc</a>.</Alert>
```

Comment:
I would try to make changes like this in separate PRs since iit gets confusing with unrelated stuff in the same diff